TileGenerator.cpp

```diff
@@ -2215,7 +2215,7 @@ void TileGenerator::renderHeightScale()
 		gdImageLine(m_image, xBorderOffset + x, yBorderOffset + 8, xBorderOffset + x, yBorderOffset + borderBottom() - 20, color.to_libgd());
 
 		int iheight = int(height + (height > 0 ? 0.5 : -0.5));
-		int iheightMaj = int(iheight / major + (height > 0 ? 0.5 : -0.5)) * major;
+		int iheightMaj = int(iheight / major + (height > 0 ? 0.5 : -0.5) * major);
```

Comment:
This changes the result of the computation.

What is the compiler complaining about ?